### ISSUE TITLE
please make fonts at least 2-3x bigger everywhere, menu, toolbars, etc. etc.

### ISSUE THREAD

**Issue by kgignatyev** [git-up/GitUp#785]
I like this tool and functionality, but everything is too small on 27' monitor, almost unreadable

**Comment by JanX2** [comment]
@kgignatyev 
Did you see the the text size slider in Preferences?

**Comment by kgignatyev** [comment]
Of course, and moved it to the biggest settings, but everything is waay too small 
![image](https://user-images.githubusercontent.com/868783/143900618-bee76e5d-1900-4a5e-b663-0a9447d1ae43.png)

**Comment by kgignatyev** [comment]
:) I tried, it is terrible. Apple is pretty bad about that IMO. That is why I see in most of applications I use there are some sort of tricks played to avoid Apple's limitations.

**Comment by lapfelix** [comment]
<img width="393" alt="image" src="https://user-images.githubusercontent.com/4634735/144262906-e5aed0b3-6813-4aa5-a7f5-f16b0a294e4b.png">
Did you try using a scaling mode on your 27" display? GitUp's text is small, but a lot of Apple's system UI is just as small so you probably want a system wide solution to this.

**Comment by kgignatyev** [comment]
this option is available on my mac display, but causes performance degradation. Not available on external 2K display, well, there is a way to change resolution but then everything is ugly. Attaching screenshot from 2K display to illustrate how small it looks compared to the texts in other applications.
![image](https://user-images.githubusercontent.com/868783/144267915-5ebfb22b-1262-4ed7-a76a-014336136384.png)

**Comment by sesam** [comment]
I also have had some issues to get any use of the font size setting; like this: whenever I show something in gitup to colleagues they want me to increase the text first. When I try, the things I'd need zoomed, like the Cmd-2 Diffs view, or viewing contents of a commit, or tooltips, seem to not get any bigger anyways.

Maybe a 2x zoom on the whole gitup app could be an option? I.e., would it be a problem if also the size of the "commit dots" and lines between those dots become bigger?

**Comment by GaryElshaw** [comment]
@lucasderraugh Lucas, is GitUp using auto-layout? My favourite music player [Cog](https://github.com/losnoco/Cog) uses a couple of things that work brilliantly for scaling font sizes. I'm one of the contributors to Transmission, and if we can get auto-layout fully working, it's my primary focus to include these.

https://github.com/losnoco/Cog/tree/main/Transformers

**Comment by lucasderraugh** [comment]
@GaryElshaw It does in a few places but not everywhere.

**Comment by prostolyubo** [comment]
> I like this tool and functionality, but everything is too small on 27' monitor, almost unreadable

If you will make this ttext bigger you will wreck it for small screens. You cant just request a change that benefits solely you and detriments others. What we should do is to try and improve the font changigng capability not straight force enlarged one on everybody.
